src/org/jitsi/impl/neomedia/transform/AbsSendTimeEngine.java

```diff
@@ -129,8 +130,66 @@ private void setTimestamp(byte[] buf, int off)
      * disable this transformer.
      * @param id the ID to set.
      */
-    public void setExtensionID(int id)
+    public void setExtensionID(byte id)
     {
         extensionID = id;
     }
+
+    /**
+     *
+     * @param packet is the RawPacket
+     * @returns the Absolute Send Time header extension if present
+     */
+    public HeaderExtension hasAbsoluteSendTimeExtension(RawPacket packet)
```

Comment:
Can we rename this to getAbsoluteSendTimeExtension? Do you need to use this from outside the class? If not, it should be private.